frontend/Screens/Home.tsx

```diff
@@ -57,10 +55,9 @@ const Home = ({navigation}) => {
     // }, [])
 
     return <View>
-        <Text>you're logged in</Text>
+        <Text>you're logged in!</Text>
         <Button onPress={() => logOut()} title={'logOut'}/>
         <Button onPress={() => navigation.navigate('Landingpage')} title={'go back to landingPage'}/>
```

Comment:
this button does not redirect me to the landing page even though I am logged in

Reply:
removed it since landing page was more of a test, kept the actual file in screen for future use.